ArduCopter/Parameters.h

```diff
@@ -292,7 +292,9 @@ class Parameters {
         k_param_radio_tuning,
         k_param_radio_tuning_high,
         k_param_radio_tuning_low,
-        k_param_rc_speed = 192,
```

Comment:
I don't like this switch-a-roo even though it's technically correct. Can't you just insert the new k_param_failsafe_gcs_timeout below k_param_radio_tuning_low? Currently k_param_radio_tuning_low == 189 and k_param_rc_speed == 192, this new one should be k_param_failsafe_gcs_timeout == 190 and since there's one empty spot there hiding then it would be nice to explicitly state "k_param_failsafe_gcs_timeout = 190," to reduce future human error.